v0/destinations/indicative/data/INIdentifyConfig.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "destKey": "uniqueId",
+    "sourceKeys": ["userId", "context.traits.userId", "context.traits.id"],
```

Comment:
should we take userId in case of identify from anywhere else except toplevel? Bcz context is modifiable by user but top level userId is always set by sdks for identify

Reply:
Agreed. This is for fail-safety given HTTP APIs can send any data. Idea is to maximize the possibilities.

Reply:
okay